src/inputs/sflow/SflowInputStream.cpp

```diff
@@ -153,6 +153,7 @@ void SflowInputStream::stop()
 void SflowInputStream::info_json(json &j) const
 {
     common_info_json(j);
+    j["input"]["packet_error"] = _error_count.load();
```

Comment:
see https://github.com/ns1labs/pktvisor/blob/b46517133d82b3ac4d8cbdfff0d78618cbeb68c7/src/inputs/pcap/PcapInputStream.cpp#L581

this needs to be prefixed with schema_key(). in that case you can leave out "input" and just make it `j[schema_key()]["packet_errors"]` (note plural)